qa/src/test/java/io/camunda/migrator/qa/history/entity/HistoryProcessInstanceTest.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.qa.history.entity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.migrator.impl.persistence.IdKeyMapper;
+import io.camunda.migrator.impl.util.ConverterUtil;
+import io.camunda.migrator.qa.history.HistoryMigrationAbstractTest;
+import io.camunda.search.entities.ProcessInstanceEntity;
+import java.util.List;
+import org.camunda.bpm.engine.history.HistoricActivityInstance;
+import org.camunda.bpm.engine.history.HistoricProcessInstance;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.camunda.bpm.engine.task.Task;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class HistoryProcessInstanceTest extends HistoryMigrationAbstractTest {
+
+  @Test
+  public void shouldMigrateProcessInstances() {
+    deployer.deployCamunda7Process("userTaskProcess.bpmn");
+
+    // given
+    ProcessInstance c7Process = runtimeService.startProcessInstanceByKey("userTaskProcessId");
+    completeAllUserTasksWithDefaultUserTaskId();
+    HistoricProcessInstance historicProcessInstance = historyService.createHistoricProcessInstanceQuery()
+        .processInstanceId(c7Process.getId())
+        .singleResult();
+
+    // when
+    historyMigrator.migrate();
+
+    // then
+    List<ProcessInstanceEntity> processInstances = searchHistoricProcessInstances("userTaskProcessId");
+    assertThat(processInstances).hasSize(1);
+    assertC8ProcessInstanceFields(processInstances.getFirst(), historicProcessInstance, "userTaskProcessId",
+        ProcessInstanceEntity.ProcessInstanceState.COMPLETED, "custom-version-tag", null, false, false);
```

Comment:
The `null` tenant in C8 would be `<default>` I believe:
```suggestion
    assertC8ProcessInstanceFields(processInstances.getFirst(), historicProcessInstance, "userTaskProcessId",
        ProcessInstanceEntity.ProcessInstanceState.COMPLETED, "custom-version-tag", "<default>", false, false);
```

Reply:
I am not sure what's the context here. I believe that multi-tenancy can be disabled in C8, then tenantId should be `null`, or I missed something?

Reply:
[Docs](https://docs.camunda.io/docs/self-managed/concepts/multi-tenancy/#the-tenant-identifier) say `The tenant identifier will be set as a property to any data produced by Camunda 8. When multi-tenancy is disabled, all data is mapped to the <default> tenant identifier.`

Reply:
We actually also have this ticket indicating multi tenancy migration is currently blocked, but I think that only relates to runtime data: https://github.com/camunda/camunda-bpm-platform/issues/5315
Maybe we can also double check this in tomorrows sync.